app/controllers/api/services_controller.rb

```diff
@@ -65,8 +64,19 @@ def destroy
     redirect_to provider_admin_dashboard_path
   end
 
-  protected
+  private
+
+  def find_backend_api
+    current_account.backend_apis.find(params[:service][:backend_api])
+  end
 
+  def service_params
+    params.require(:service).permit(:system_name, :name, :description, :act_as_product).tap do |p|
+      p[:backend_api_configs] = [ BackendApiConfig.new(backend_api: find_backend_api) ] if params[:service][:backend_api].present?
```

Comment:
only do this is the account is authorized to do it :smile: or if we are not doing the authorizations in depth yet, then at leaaast make sure the rolling update is enabled

Reply:
and add a test to make sure this backend api part works and only if the rolling update is enabled